dependencies.md

```diff
@@ -1,4 +1,20 @@
-# Composer log
+# System Requirements
+
+* A webserver like Apache or Nginx
+* PHP from 5.4
```

Comment:
PHP 5.4 may not be the right version, because one dependency (`friendsofvictoire/button-widget`) require PHP 5.5.9. See this comment for further explanation: https://github.com/Victoire/victoire/commit/f4c8e427e551b0a1501bb23e654bb1f12755bca0#commitcomment-20861764